stories/Components/Dropdown.stories.jsx

```diff
@@ -353,6 +355,38 @@ const CustomDropdown = args => {
 
 CustomDropdown.storyName = "Custom Dropdown";
 
+const EventBubblingAndCapturing = args => {
+  const { Menu, MenuItem, Divider } = Dropdown;
+  const listItems = ["Action", "Another action", "Something else here"];
+
+  return (
+    <div className="h-40" onClick={event => event.stopPropagation()}>
+      <Dropdown label="Dropdown" {...args}>
+        <Menu>
+          {listItems.map((item, idx) => (
+            <MenuItem.Button key={idx}>{item}</MenuItem.Button>
+          ))}
+          <MenuItem.Button isActive>Active</MenuItem.Button>
+          <MenuItem.Button isDisabled>Disabled</MenuItem.Button>
+          <Divider />
+          <MenuItem.Button style="danger">Delete</MenuItem.Button>
+        </Menu>
+      </Dropdown>
+    </div>
```

Comment:
Let's update the story with a container (like a card) that has an onClick handler, to demonstrate that the container onClick event handler is not triggered on clicking the dropdown